birdhouse/deployment/triggerdeploy.sh

```diff
@@ -34,7 +34,7 @@
 #   Follow same instructions in deploy.sh.
 
 if [ ! -z "$AUTODEPLOY_SILENT" ]; then
-    LOG_FILE="/var/log/PAVICS/autodeploy.log"
+    LOG_FILE="/var/log/birdhouse/autodeploy.log"
```

Comment:
Yes... but it's tricky because we can't get that variable until we source the configs and we don't get those until a bunch of code has already run in which case the logs would be lost. I'll add a suggested fix but I'd like to get @tlvu 's opinion as well

Reply:
Misha is probably right.  I will have to get back to you guys once I try this PR for real on my VM.